### PR TITLE
u-boot-tools 2020.04

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/u-boot-tools.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class UBootTools < Formula
   desc "Universal boot loader"
   homepage "https://www.denx.de/wiki/U-Boot/"
-  url "https://ftp.denx.de/pub/u-boot/u-boot-2020.01.tar.bz2"
-  sha256 "aa453c603208b1b27bd03525775a7f79b443adec577fdc6e8f06974025a135f1"
+  url "http://ftp.denx.de/pub/u-boot/u-boot-2020.04.tar.bz2"
```
Comment (Bo98) [inline Formula/u-boot-tools.rb:4]: This should use HTTPS

Comment (chenrui333) [reply Formula/u-boot-tools.rb]: 👍

**Comment by SMillerDev** [comment]
https://github.com/Homebrew/homebrew-core/pull/53132/checks?check_run_id=586027736#step:4:281

**Comment by Bo98** [comment]
This is one of these "I'm not sure how this ever compiled on macOS"

**Comment by chenrui333** [comment]
Closing as not actively working on.

**Pull request closed without merging**